content/docs/tooling/avalanche-cli.mdx

```diff
@@ -2,1228 +2,4180 @@
+the --output flag.
+- [`import`](#blockchain-import): Import blockchain configurations into avalanche-cli.
+
+This command suite supports importing from a file created on another computer,
+or importing from blockchains running public networks
+(e.g. created manually or with the deprecated subnet-cli)
+- [`join`](#blockchain-join): The subnet join command configures your validator node to begin validating a new Blockchain.
+
+To complete this process, you must have access to the machine running your validator. If the
+CLI is running on the same machine as your validator, it can generate or update your node's
+config file automatically. Alternatively, the command can print the necessary instructions
+to update your node manually. To complete the validation process, the Subnet's admins must add
+the NodeID of your validator to the Subnet's allow list by calling addValidator with your
+NodeID.
+
+After you update your validator's config, you need to restart your validator manually. If
+you provide the --avalanchego-config flag, this command attempts to edit the config file
+at that path.
+
+This command currently only supports Blockchains deployed on the Fuji Testnet and Mainnet.
+- [`list`](#blockchain-list): The blockchain list command prints the names of all created Blockchain configurations. Without any flags,
+it prints some general, static information about the Blockchain. With the --deployed flag, the command
+shows additional information including the VMID, BlockchainID and SubnetID.
+- [`publish`](#blockchain-publish): The blockchain publish command publishes the Blockchain's VM to a repository.
+- [`removeValidator`](#blockchain-removevalidator): The blockchain removeValidator command stops a whitelisted, subnet network validator from
+validating your deployed Blockchain.
+
+To remove the validator from the Subnet's allow list, provide the validator's unique NodeID. You can bypass
+these prompts by providing the values with flags.
+- [`stats`](#blockchain-stats): The blockchain stats command prints validator statistics for the given Blockchain.
+- [`upgrade`](#blockchain-upgrade): The blockchain upgrade command suite provides a collection of tools for
+updating your developmental and deployed Blockchains.
+- [`validators`](#blockchain-validators): The blockchain validators command lists the validators of a blockchain's subnet and provides
+several statistics about them.
+- [`vmid`](#blockchain-vmid): The blockchain vmid command prints the virtual machine ID (VMID) for the given Blockchain.
+
+### addValidator
+
+The blockchain addValidator command whitelists a primary network validator to
+validate the subnet of the provided deployed Blockchain.
+
+To add the validator to the Subnet's allow list, you first need to provide
+the blockchainName and the validator's unique NodeID. The command then prompts
+for the validation start time, duration, and stake weight. You can bypass
+these prompts by providing the values with flags.
```

Comment:
Are the descriptions word-for-word from the CLI, or generated by ai? If this page is generated from CLI master, this is still accurate since L1 language should come with cli changes

Reply:
descriptions are word for word from the CLI